signatures/rego/regosig/aio_test.go

```diff
@@ -6,9 +6,11 @@ import (
 	"os"
 	"testing"
 
+	"github.com/aquasecurity/tracee/signatures/rego/regosig"
```

Comment:
I'm not sure we should move aio.go and traceerego.go. These for me belong to tracee-rules core functionality and just rely on Rego file that we are trying to move to it's own location under ./signatures/rego

/cc @itaysk WDYT?

Reply:
agree!

Reply:
also aio.rego, and probably mapper.go? what is Mapper?

Reply:
- aio.rego is integral part of aio.go (actually Go-embedded)
- mapper.go is utility code shared between aio.go and traceerego.go (mapping of Rego eval results to tracee-rules structs)